Add `configDescription` field for plugins (#747)
wulkano/kap: docs/plugins.md

The share service is a plain object defining some metadata:

- `title`: The title used in the export menu. For example: `Share to GIPHY`.<br>The text should be in [title case](https://capitalizemytitle.com), for example, `Save to Disk`, not `Save to disk`.

> - `description`: A description that shows up at the top of the configuration window. You can use this to explain the config options or link to API docs. Any links in this description will be parsed into clickable links automatically.

Reviewer: ```suggestion
- `description`: A description displayed at the top of the configuration window. You can use this to explain the config options or link to API docs. URLs in the description will automatically be linkified.
```

?

Reviewer: Not sure about `description` as a name. That sounds more like it's the description of the plugin, rather than its config.

Author: `configDescription`? It's on the shareService object

Reviewer: 👍
